local/build-binary

```diff
@@ -171,6 +176,7 @@ pushd ${WORKDIR}
         -DWITH_INNODB_MEMCACHED=ON \
         -DENABLE_DOWNLOADS=ON \
         -DDOWNLOAD_ROOT=${DOWNLOAD_DIR} \
+        -DWITH_KEYRING_VAULT=${WITH_KEYRING_VAULT} \
         -DDOWNLOAD_BOOST=ON \
         -DWITH_BOOST=${DOWNLOAD_DIR} \
         -DWITH_KEYRING_VAULT=ON \
```

Comment:
@hors does it actually work? This results in "-DWITH_KEYRING_VAULT=OFF ... -DWITH_KEYRING_VAULT=ON"